config/config.exs

```diff
@@ -18,4 +18,7 @@ if Mix.env() == :test do
     sasl_error_logger: false
 
   config :tesla, MockClient, adapter: Tesla.Mock
+
+  config :tesla, Tesla.Adapter.Mint,
+    cacert: ["./deps/httparrot/priv/ssl/server-ca.crt"]
```

Comment:
@ericmj This config won't end up in production builds, right? I mean, it will be `nil` by default so it should be safe to reference httparrot here.

Reply:
I think it will be ```nil``` by default.

Reply:
Correct, `config.exs` is not used from dependencies. So the `Tesla.Adapter.Mint` key will not be set.